docs/layers.md

```diff
@@ -524,6 +526,7 @@ _TIP: Some `landuse` features only exist as point features in OpenStreetMap. Fin
 * `groyne`
 * `guard_rail`
 * `hanami`
+* `harbor` (a.k.a harbour)
```

Comment:
Also fix the documentation spelling (to `harbour` without the parenthetical note)